docs/specs/D-kit/Camera_Based_Auto_Driving/Start_Auto_Driving_cn.md

```diff
@@ -69,6 +69,8 @@
 | `/apollo/sensor/camera/front_6mm/image` | 确保能正常输出数据、帧率在15帧左右|
 |`/tf`|确保能正常输出数据|
 |`/tf_static`|确保能正常输出数据|
+|`/apollo/canbus/chassis`|确保能正常输出数据|
+|`/apollo/canbus/chassis_detail`|确保能正常输出数据|
```

Comment:
1. Generate Virutal Lane is also a prerequisite
2.Launch Camera perception from dreamview.